src/main/java/racingcar/resultview/ConsoleResultView.java

```diff
@@ -0,0 +1,21 @@
+package racingcar.resultview;
+
+import racingcar.domain.RacingCar;
+
+import java.util.List;
+
+public class ConsoleResultView implements ResultView {
+    private static final String PROGRESS_BAR = "-";
+
+    @Override
+    public void print(List<RacingCar> cars) {
+        StringBuilder stringBuilder = new StringBuilder();
+
+        for (RacingCar car : cars) {
+            car.repeatAsMoveCount(() -> stringBuilder.append(PROGRESS_BAR));
+            stringBuilder.append("\n");
```

Comment:
OS 마다 줄바꿈을 인식하는게 다르기 때문에 jdk 1.7부터 도입된 현재 OS에 맞는 줄바꿈을 얻어오도록 하면 어떨까요?
```suggestion
            stringBuilder.append(System.lineSeparator());
```